pages/dkp/kommander/2.1/release-notes/2.1.0/index.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: layout.pug
+navigationTitle: Kommander 2.1.0 Release Notes
+title: Kommander 2.1.0 Release Notes
+menuWeight: 10
+excerpt: View release-specific information for Kommander 2.1.0
+enterprise: false
+beta: false
+---
+
+**D2iQ&reg; Kommander&reg; version 2.1.0 was released on November 19, 2021.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Kommander[/button]
+
+To get started with Kommander, [download](../download/) and [install](../install/) the latest version of Kommander.
```

Comment:
This used to say Konvoy?? We should check to see what is REALLY the process now for Kommander.